src/solver/lu_decomposition.cu

```diff
@@ -85,6 +87,11 @@ namespace micm
               size_t A_idx = d_aki[aki_offset++] + tid;
               d_L[L_idx] = d_A[A_idx];
             }
+            else {
+              size_t L_idx = d_lki_nkj[lki_nkj_offset].first + tid;
+              d_L[L_idx] = 0;
```

Comment:
The same line is found in `line 86`. I think we can define it outside the conditional